content/docs/start/data-management/data-versioning.md

```diff
@@ -0,0 +1,260 @@
+---
+title: 'Get Started: Data Versioning'
+description: 'Get started with data and model versioning in DVC. Learn how to
+use a regular Git workflow for datasets and ML models, without storing large
+files in Git.'
+---
+
+# Get Started: Data Versioning
```

Comment:
Nothing changed below, this comes straight from the existing [Data Management index page](https://dvc.org/doc/start/data-management) (after **Expand to get an example dataset**).

Reply:
Just not sure whether to call this Data **and Model** Versioning. The next page is called "Data and Model Access" so it would get a little repetitive (in the nav at least). The next one is just "Data Pipelines".